### ISSUE TITLE
Feature request: codedark_transparent

### ISSUE THREAD

**Issue by theroncross** [tomasiser/vim-code-dark#78]
related to #71 

It would be nice to have a flag to set transparent backgrounds, similar to the `codedark_conservative`. From what I can see, we'd need at least
```
 hi Normal       ctermbg=none  guibg=none
 hi CursorLineNr               guibg=none
 hi EndOfBuffer                guibg=none
 hi Folded                     guibg=none
 hi LineNr       ctermbg=none  guibg=none
 hi SignColumn   ctermbg=none  guibg=none
```
I have `g:signcolumn = 'number'`, which works with the above, but was unable to get a separate sign column text to have a transparent background.

**Comment by tomasiser** [comment]
Hi! This sounds like a good idea. Would you be up for making a PR with that change? Also not sure if `nonText` also needs to be set as in #71.

**Comment by agjs** [comment]
Any progress on this? I'd actually like to give a hand with this if @theroncross isn't done yet?

**Comment by theroncross** [comment]
Fell off my radar - please go for it. Thanks.

**Comment by tomasiser** [comment]
Hi everyone, I think this can now be closed since we have #102 merged! Thank you!
